hyperwallet/tests/test_api.py

```diff
@@ -2189,6 +2192,80 @@ def test_list_webhooks_success(self, mock_get):
 
         self.assertTrue(response[0].token, self.data.get('token'))
 
+    '''
+
+    Upload Documents
+
+    '''
+    def test_uploadDocumentsForUser_fail_need_user_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.uploadDocumentsForUser()
+
+        self.assertEqual(exc.exception.message, 'userToken is required')
+
+    @mock.patch('hyperwallet.utils.ApiClient._makeRequest')
+    def test_uploadDocumentsForUser_success(self, mock_put):
+
+        mock_put.return_value = self.data
+        response = self.api.uploadDocumentsForUser('token', self.value)
+
+        self.assertTrue(response.token, self.data.get('token'))
+
+    '''
+
+    Transfer Refunds
+
+    '''
+
+    @mock.patch('hyperwallet.utils.ApiClient._makeRequest')
+    def test_transfer_refunds_success(self, mock_post):
+
+        mock_post.return_value = self.data
+        response = self.api.createTransferRefund('token',self.data)
+        self.assertTrue(response.token, self.data.get('token'))
+
+    def test_transfer_refunds_fail_need_payment_token(self):
+
+        with self.assertRaises(HyperwalletException) as exc:
+            self.api.createTransferRefund()
+
+        self.assertEqual(exc.exception.message, 'transferToken is required')
+
+    def test_transfer_refunds_fail_need_data(self):
```

Comment:
Please follow the convention of method names. This should be test_create_transfer_refunds.... Dont forget to have the type of operation(Create/get/list/update) mentioned in the method's name.

Reply:
method name has been changed.